plugin/niassa+pinery/src/main/java/ca/on/oicr/gsi/shesmu/pinery/PineryIUSValue.java

```diff
@@ -74,12 +74,12 @@ public PineryIUSValue(//
 		this.completed_date = completed_date;
 	}
 
-	@Export(type = "d")
+	@ShesmuVariable(type = "d")
```

Comment:
Do `type = "s"` and `type = "d"` still need to be explicitly declared? (They're also declared in `InnerTestValue` and `TestValue` below)
Could it please be Javadoc'd somewhere which types need to be explicitly declared?

Reply:
No, but it does a check that if you specify the type, it matches the actual return type, so I'm leaving thsese in for testing purposes.

Reply:
I can see the value in leaving them in `InnerTestValue` and `TestValue` for testing purposes, but I'm concerned that leaving them in `PineryIUSValue` makes for potentially confusing inconsistent styles between the plugin code files.

Reply:
Ooops. Didn't remember that one. Fixed.